src/containers/Board/InfinityScrollBoard/InfinityScrollBoard.tsx

```diff
@@ -0,0 +1,60 @@
+/*
+ * Created on Wed Nov 15 2023
+ *
+ * Copyright (c) 2023 Your Company
+ */
+
+import { BoardGrid, GridItem, Wrap } from "./emotion";
```

Comment:
```suggestion
import * as S from "./emotion";
```
S는 styled-component를 의미한다고 생각하고 <S.BoardGrid> 이런식으로 사용해요